src/main/js/controls/Graph/Graph.js

```diff
@@ -247,6 +247,7 @@ class Graph extends Construct {
         ) {
             createDateline(this.scale, this.config, this.svg);
         }
+        createDefs(this.config, this.svg);
```

Comment:
`defs` is usually one of the first elements created within an SVG. Lets not break away from that tradition.

Reply:
As in mail if we keep the call on the first place then if we load the graph without any content then since the createDateline call is below createDefs call, so we can not get the element of dateline-point at the time of createDefs call which we need to do the manipulation.

Reply:
Creation can always be done first, the attributes of `rect` can be updated when the indicator has been added.

Reply:
ok, then inside the createDateline I have to update the attributes of defs right?